### DISCUSSION TITLE
FrankenPHP for laravel octane support

### DISCUSSION THREAD

**Discussion by jsmit99** [beyondcode/herd-community#1175] [Ideas]
It would be awesome if laravel herd would support laravel octane out of the box.

**Comment by nikspyratos** [comment]
Feel like I'm going crazy, swear I saw a social media post recently hinting at this? @mpociot these would be some decent minimal ways to support using Octane with Herd:

- Maybe supporting frankenphp would save needing to add swoole to the php binaries for using octane
- A small UI convenience for `herd unlink && herd proxy YOUR_APP_URL.test http://127.0.0.1:8000 --secure` (or whatever configured octane port)

**Comment by SteBram** [comment]
I've been looking through the docs, the issues and discussion here, but I cannot find any info if there's progress on this or not?

Is it planned to support Laravel Octane in any way (frankenphp for example)? I would love to be able to reproduce our production environments better this way.
